src/sql/workbench/services/accountManagement/browser/accountDialog.ts

```diff
@@ -193,9 +195,39 @@ export class AccountDialog extends Modal {
 				for (const addedProvider of addedProviders) {
 					this.addProvider(addedProvider);
```

Comment:
is there a way for the github provider to get added here, rather than separately in the `then()` below?

Reply:
In the body of the loop or below it? I'm not entirely sure. I wasn't able to locate an account object, so I had to go and find sessions for Copilot. I could move the call to `addGitHubCopilotProvider` below this loop to get rid of the second `then`.